packages/api/endpoints/execution-status.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
 const router = require('express-promise-router')();
+const moment = require('moment');
 const { getStateMachineArnFromExecutionArn } = require('@cumulus/message/Executions');
 const { pullStepFunctionEvent } = require('@cumulus/message/StepFunctions');
+const S3ObjectStore = require('@cumulus/aws-client/S3ObjectStore');
+const { s3 } = require('@cumulus/aws-client/services');
+const { buildS3Uri } = require('@cumulus/aws-client/S3');
```

Comment:
We can avoid importing `@cumulus/aws-client/services` if we import `s3PutObject` function and pass in `ACL: undefined` to `s3PutObject`
```suggestion
const { buildS3Uri, s3PutObject } = require('@cumulus/aws-client/S3');
```